src/libraries/System.Text.Json/tests/Serialization/CustomConverterTests/CustomConverterTests.NullableTypes.cs

```diff
@@ -160,32 +160,35 @@ public override void Write(Utf8JsonWriter writer, int? value, JsonSerializerOpti
         }
 
         [Fact]
-        public static void NullableConverterIsPassedNull()
+        public static void NullableConverterIsNotPassedNull()
         {
+            // For compat, deserialize does not call converter for null token unless the type doesn't support
+            // null or HandleNull is overridden and returns 'true'.
+            // For compat, serialize does not call converter for null unless null is a valid value and HandleNull is true.
```

Comment:
Are these two sentences duplicates?

Reply:
No - one applies to deserialize, and the other to serialize.

Reply:
Ops, I missed that.